src/main/java/com/spotify/folsom/client/Utils.java

```diff
@@ -76,4 +76,11 @@ public static int ttlToExpiration(final int ttl) {
     };
   }
 
+  /**
+   * A counter of all currently connected clients. This can be useful to detect connection leaks
+   */
+  public static int getGlobalConnectionCount() {
+    return DefaultRawMemcacheClient.GLOBAL_CONNECTION_COUNT.get();
```

Comment:
How about exposing this as part of Metrics?

Reply:
I will look into it!